### ISSUE TITLE
better looking default template when ng new

### ISSUE THREAD

**Issue by vthinkxie** [angular/angular-cli#14946]
# 🚀 Feature request

### Command (mark with an `x`)

```
- [x] new
- [ ] build
- [ ] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc
```

### Description

Angular CLI should provide a better looking default template, and the current template is a bit rudimentary

### Describe the solution you'd like

The create-react-app default UI
![FireShot Capture 341 - React App - localhost](https://user-images.githubusercontent.com/1506722/60399079-1157ad80-9b92-11e9-9165-79ca4730eb06.png)

The vue-cli default UI
![FireShot Capture 343 - hello-world - localhost](https://user-images.githubusercontent.com/1506722/60399081-187ebb80-9b92-11e9-9f53-00ae0b726bde.png)

The angular-cli default UI
![FireShot Capture 342 - NgTest - localhost](https://user-images.githubusercontent.com/1506722/60399082-1f0d3300-9b92-11e9-82a0-a90fff99a4d6.png)

### Describe alternatives you've considered
 Have you considered any alternative solutions or workarounds?

The default UI when ng new should have some default style to make it looks better.

**Comment by alan-agius4** [comment]
Will be addressed by https://github.com/angular/angular-cli/pull/14403

**Comment by alan-agius4** [comment]
Closing as this has been addressed by https://github.com/angular/angular-cli/pull/14403 and will be available in 8.3
